src/test/kotlin/no/nav/syfo/service/DialogmoteInnkallingVarselServiceSpek.kt

```diff
@@ -127,25 +157,268 @@ object DialogmoteInnkallingVarselServiceSpek : Spek({
                     any(),
                     dialogmoteInnkallingVarselService.getVarselUrl(varselHendelse, journalpostUuid),
                     dialogmoteInnkallingVarselService.getMeldingTypeForSykmeldtVarsling(hendelseType),
-                    false
+                    false,
                 )
             }
             verify(exactly = 0) {
                 fysiskBrevUtsendingService.sendBrev(
                     journalpostUuidAddressProtection,
-                    journalpostIdAddressProtection
+                    journalpostIdAddressProtection,
+                )
+            }
+            verify(atLeast = 1) {
+                dittSykefravaerMeldingKafkaProducer.sendMelding(
+                    any(),
                 )
             }
         }
+
+        it("Users should not be notified when lest hendelse is sent") {
+            coEvery { accessControlService.getUserAccessStatus("fnr1") } returns
+                UserAccessStatus("fnr1", true, true)
+
+            coEvery { dittSykefravaerMeldingKafkaProducer.ferdigstillMelding(any(), any()) } returns Unit
+            coEvery { brukernotifikasjonerService.ferdigstillVarsel(any(), any()) } returns Unit
```

Comment:
Ser ikke ut som disse er coroutines, så da er det greit å bytte til every i stedet for coEvery :)